src/main/scala/nl.knaw.dans.easy.validatebag/package.scala

```diff
@@ -33,6 +35,8 @@ package object validatebag {
     1 -> "doi:10.17026/dans-zf3-q54n"
   )
 
+  val licensesDir = Paths.get("target/easy-licenses/licenses")
```

Comment:
This is not correct. This may be right while working on the project, but not after you compiled it and running it on deasy. Use `getClass.getResource(...)` instead.
Also inline this in the `Configuration` class instead.

Reply:
You are right.  But now that a mapping was added into building the rpm, getClass.getResource is not needed anymore.  Licence directory is now in the `home` directory